src/xcode/ENA/ENA/Source/Scenes/ContactDiary/DiaryService.swift

```diff
@@ -110,12 +110,14 @@ class MockDiaryStore: DiaryStoring {
 
 	func removeContactPerson(id: Int) {
 		contactPersons.removeAll { $0.id == id }
+		contactPersonEncounters.removeAll { $0.contactPersonId == id }
```

Comment:
Since I can't comment at the top of the File I have to do it here.
Nice to see a Mock, would be even nicer, if it lived inside its own File :D

Reply:
We will move these models all into separate files once the actual store implementation is done :)